core/src/main/java/com/exadel/kaliada/core/models/LikeModel.java

```diff
@@ -0,0 +1,30 @@
+package com.exadel.kaliada.core.models;
+
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.models.annotations.Default;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.ValueMapValue;
+
+@Model(adaptables = SlingHttpServletRequest.class,
+        resourceType = LikeModel.RESOURCE_TYPE,
+        defaultInjectionStrategy = DefaultInjectionStrategy.REQUIRED)
+public class LikeModel {
+    protected static final String RESOURCE_TYPE = "aemdemo/components/like";
+
+    @ValueMapValue
+    @Default(intValues = 0)
```

Comment:
Please always prefer primitives over boxed values. Therefore it's `long`.
In this case, `@Default` is not needed because will `=0` just from the start